### PR TITLE
Restructured Settings and Database file

### PR DESCRIPTION
**Pull request by iamaniekan**
Changes were made to the `settings.py` and `database.py` files for proper structures.

### CODE REVIEW

File: api/v1/schemas/subscription.py
Comment (bolexs) [inline api/v1/schemas/subscription.py:1]: can this file be ignored

File: api/v1/routes/subscription.py
Comment (bolexs) [inline api/v1/routes/subscription.py:1]: @iamaniekan  this as well

Comment (iamaniekan) [reply api/v1/routes/subscription.py:1]: The files are empty

Comment (bolexs) [reply api/v1/routes/subscription.py:1]: please add them to .gitignore

Comment (iamaniekan) [reply api/v1/routes/subscription.py:1]: Okay noted

**Comment by bolexs** [comment]
this was approved to make the codebase cleaner

**Pull request merged**